pcsx2/gui/Panels/GSWindowPanel.cpp

```diff
@@ -131,7 +150,9 @@ void Panels::GSWindowSettingsPanel::ApplyConfigToGui( AppConfig& configToApply,
 		m_check_SizeLock	->SetValue( conf.DisableResizeBorders );
 
 		m_combo_AspectRatio	->SetSelection( (int)conf.AspectRatio );
+		m_combo_AspectRatio	->Enable( conf.ScalingType != ScalingType_Fill );
```

Comment:
Missing [bind event handler](http://docs.wxwidgets.org/3.1.0/classwx_evt_handler.html#a0f30c8fa5583b4a5f661897d63de3b62) for the Aspect ratio combobox grayout?